addons/binding/org.openhab.binding.network/README.md

```diff
@@ -147,17 +147,17 @@ Things support the following channels:
 
 ## Full Example
 
-demo.Things:
+demo.things:
 
 ```xtend
-network:pingdevice:devicename [ hostname="192.168.0.42" ]
+Thing network:device:devicename [ hostname="192.168.0.42" ]
```

Comment:
imho it should be device in this version of the code, are you sure that it should be device?